docs/operate/mobility/move-base.md

```diff
@@ -1,9 +1,28 @@
 ---
 linkTitle: "Move a base"
-title: "Move a base through space"
+title: "Move a wheeled robot base"
 weight: 40
 layout: "docs"
 type: "docs"
 no_list: true
 description: "Move a mobile robot with manual or autonomous navigation."
 ---
+
+You have three options for moving a mobile robot [base](/operate/reference/components/base/):
```

Comment:
Guessing this isn't done yet but make sure this has some example code on this page please. Possible that both the pages drive a rover and navigate with a rover should just be in this page instead.

Reply:
Do you want both this and the arm page to follow the pattern that's now in place on the gantry page, just with the addition of links to the tutorials somewhere in there? I think for base and arm it still makes sense to keep the tutorials because there's complexity and because we have the tutorials available that take someone from start to finish.

Reply:
I've decided to try putting the how-to arm API content into the page, making it not an index page anymore. I put the more complex tutorials back into unlisted tutorials and linked to them from the new Use automated complex motion planning section in Arm. I still want to shorten those but for now at least this puts some code samples onto the arm page, and makes it mostly fit the format of gantry. Same can happen with base, just put the how-to into this page, and link to longer service tutorials for now.